var/spack/repos/builtin/packages/acl/package.py

```diff
@@ -27,7 +27,8 @@ class Acl(AutotoolsPackage):
     depends_on("gettext")
 
     def setup_build_environment(self, env):
-        env.append_flags("LDFLAGS", "-lintl")
+        if "intl" in self.spec["gettext"].libs.names:
+            env.append_flags("LDFLAGS", "-lintl")
```

Comment:
Why not use:
```suggestion
        env.append_flags("LDFLAGS", self.spec["gettext"].libs.link_flags)
```

Reply:
If `libs.link_flags` gets filled in properly for externals that would be a reasonable thing to do.

Reply:
It will add links for:
```python
["libasprintf", "libgettextlib", "libgettextpo", "libgettextsrc", "libintl"]
```

Reply:
It appears that, of ackages requiring `libintl`, most of them require only that and not any of the others. Indeed, putting the full list in for `elfutils` actually caused an ODR violation as `color_mode` is defined in `libeu.a` and `libgettextsrc` (IIRC). In other words, using only `-lintl` when needed, rather than the full `libs.link_flags`, seems generally to be the way to go.